cluster/saltbase/salt/kube-addons/kube-addons.sh

```diff
@@ -24,6 +24,18 @@ ADDON_CHECK_INTERVAL_SEC=${TEST_ADDON_CHECK_INTERVAL_SEC:-600}
 SYSTEM_NAMESPACE=kube-system
 token_dir=${TOKEN_DIR:-/srv/kubernetes}
 
+PYTHON=python
```

Comment:
Do you actually want to set this before `check_python` is called? If you leave it unbound, `check_python` will be required due to `nounset`. The way it's set right now, you don't have to call `check_python` at all.